sklearn/model_selection/_validation.py

```diff
@@ -77,7 +77,7 @@ def _check_params_groups_deprecation(fit_params, params, groups, version):
         )
         params = fit_params
 
-    params = {} if params is None else params
+    params = params or {}
```

Comment:
Why change? The original is more specific in only replacing `None`, and not things like `{}` and the various other things that are false-y.

Reply:
it's always a dictionary anyway

Reply:
I don't understand this answer. If the user didn't pass any `params` then we want to replace `None` with `{}`. Which in Python is `{} if params is None else params`, so I don't understand why you want to make the change to something that is less precise in expressing the intention